dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -2277,25 +2277,9 @@ public void executeSparkJob(Vector jobClassNames, String logFileName) {
             SnappyBB.getBB().getSharedMap().put("START_RANGE_APP2", finalEnd2 + 1);
             SnappyBB.getBB().getSharedMap().put("END_RANGE_APP2", finalEnd2 + 100);
           } else if (appName.equals("CDCIngestionApp1")) {
-            userAppArgs = userAppArgs + " " + finalStart + " " + finalEnd;
+            userAppArgs = finalStart + " " + finalEnd + " " + userAppArgs;
```

Comment:
The CDC code can be moved to a method and the method call be called here.

Reply:
Done